### PR TITLE
SEO and markup edits

### PR DESCRIPTION
**Pull request by scottmathson**
work-in-progress pr/branch for Scott to add metadata and optimize markup throughout, prior to launch

### CODE REVIEW

**Comment by scottmathson** [comment]
fyi @philhawksworth this series of commits here adds trailing slashes so as to resolve to a `200` right away versus `301` happening due to missing `/` - this helps when bots crawl to pass page equity across directly and avoid redirect chains.

trailing slash commits
ex: https://github.com/jamstack/jamstack.org/pull/361/commits/14980a4bc93ac68823ec34888311cdb620b6eb7f

File: src/site/_includes/components/generator-card.njk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div id="{{item.id}}" class="generator-card text-left p-4 mb-8 text-gray-700 bg-white rounded">
   <h3 class="text-xl font-bold bg-teal-600">
-    <a href="{{ item.data.title | slug}}" >{{ item.data.title }}</a>
+    <a href="{{ item.data.title | slug}}/" >{{ item.data.title }}</a>
```
Comment (zachleat) [inline src/site/_includes/components/generator-card.njk:3]:
```
<a href="{{ item.url }}">{{ item.data.title }}</a>
```

would be better here (and will have the trailing slash)

File: src/site/_includes/components/cms-card.njk [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div id="{{item.id}}" class="cms-card text-left p-4 mb-8 text-gray-700 bg-white rounded">
   <h3 class="text-xl font-bold bg-orange-500">
-    <a href="{{ item.data.title | slug}}" >{{ item.data.title }}</a>
+    <a href="{{ item.data.title | slug}}/" >{{ item.data.title }}</a>
```
Comment (zachleat) [inline src/site/_includes/components/cms-card.njk:3]:
```html
<a href="{{ item.url }}">{{ item.data.title }}</a>
```

same as the other, would be better here (and will have the trailing slash)

File: .eleventy.js [hunk 3]
```diff
@@ -102,6 +102,9 @@ module.exports = function(eleventyConfig) {
   eleventyConfig.addFilter('dumpasyaml', obj => {
     return yaml.safeDump(obj)
   });
+  
+  // add site alias for base layout
+  eleventyConfig.addLayoutAlias('site', 'layouts/base.njk');
```
Comment (zachleat) [inline .eleventy.js:107]: I don’t know if we need this? Doesn’t seem to be used anywhere

**Review by zachleat — CHANGES_REQUESTED** [review]
Just a few minor suggestions

File: src/site/_includes/components/generator-card.njk (hunk 1 above)
Comment (scottmathson) [reply src/site/_includes/components/generator-card.njk:3]: ohhh very nice, thanks!

File: .eleventy.js (hunk 3 above)
Comment (scottmathson) [reply .eleventy.js:107]: using in base.njk - not familiar with config setup here, but used so as to reference `site.title` in `base.njk`

Comment (scottmathson) [reply .eleventy.js]: https://github.com/jamstack/jamstack.org/pull/361/commits/d6034ebfc68023855d73902101207dd9e5a166a1
https://github.com/jamstack/jamstack.org/pull/361/commits/0f1b3d31236b75edb3dd5c05ad32a26ae030d917
https://github.com/jamstack/jamstack.org/pull/361/commits/eb4ffe6113b69fc8ed2e36d31c5e3911c9ae124c

**Review by philhawksworth — APPROVED** [review]

**Pull request merged**